docs/source/contrib_guide_docs.rst

```diff
@@ -46,7 +46,11 @@ we recommend that you install the stable development version Sphinx
 (``pip install git+https://github.com/sphinx-doc/sphinx@stable``) or the
 current released version of Sphinx (``pip install sphinx``).
 
-The following commands should be executed using the Terminal/command line from
+In addition, you may need the following packages: sphinxcontrib-spelling, sphinx_rtd_theme, and pyenchant, which can be installed via ``pip install sphinxcontrib-spelling sphinx_rtd_theme pyenchant``.
+
+If you are on Linux, the last command will fail if you don't have the Enchant C library, and you will need to run ``sudo apt-get install enchant``.
```

Comment:
The installation itself doesn't fail - pyenchant has a pure Python wheel on PyPI, so it doesn't try to build or load enchant at install time. Presumably it will fail when you try to use it, but I can't check that because when I try to remove it, it uninstalls about 100 dependent packages.